### ISSUE TITLE
Not working on Node 12

### ISSUE THREAD

**Issue by BerniWittmann** [apideck-libraries/portman#174]
The cli is currently not working with node 12, as the `openapi-format` introduced changes that made itself incompatible with node 12 (right now)

Related issue for reference: https://github.com/thim81/openapi-format/issues/21

This is only happening when a filterFile is specified and results in an error `Error: Parsing ./tmp/converted/filtered.yml failed.` as `filtered.yml` is not created by `openapi-format`

Just wanted to make you aware of the issue, that you might have to upgrade the dependency of `openapi-format` once it has fixed this.

Workaround for now: Either do not use filtering, or ensure portman is run on node 14

**Comment by thim81** [comment]
@BerniWittmann I responded to the issue reported on openapi-format. I ll try to provide a fix in the openapi-format package which will solve this issue. 

Thanks again for reporting it, it is appreciated.

**Comment by BerniWittmann** [comment]
@thim81 thank you, I didn't realize that you were a maintainer in both repositories :)

**Comment by thim81** [comment]
@BerniWittmann We just released 1.10.2, which should fix your issue reported.

We bumped the openapi-format dependency to the latest version, which handle Node 12 better.

For your awareness: If you are using a Node version, lower than Node 14, Portman will not use "optional chaining" to generated Postman collection.

Given that Node 12 is in maintenance mode until mid 2022, the tip is to start using at least Node 14 or higher.

https://nodejs.org/en/about/releases/
![image](https://user-images.githubusercontent.com/952446/143837757-aaf33e86-b968-4bd2-878d-0cc86b2e9c49.png)
